forms-flow-web/src/components/ServiceFlow/list/sort/CreateNewFilter.js

```diff
@@ -86,6 +88,8 @@ export default function CreateNewFilterDrawer({
   const [specificUserGroup, setSpecificUserGroup] = useState("");
   const firstResult = useSelector((state) => state.bpmTasks.firstResult);
   const tenantKey = useSelector((state) => state.tenants?.tenantId);
+  const process = useSelector((state) => state.process?.processList);
+  const processList = listProcess(process, true);
```

Comment:
cache this processList 
`const processList = useMemo(
    () => listProcess(process, true),
    [process]
  );
`